axelor-account/src/main/java/com/axelor/apps/account/service/ReimbursementImportService.java

```diff
@@ -79,11 +84,13 @@ public void runReimbursementImport(Company company) throws AxelorException, IOEx
     this.testCompanyField(company);
 
     AccountConfig accountConfig = company.getAccountConfig();
+    String dataUploadDir = appService.getFileUploadDir();
+    dataUploadDir = dataUploadDir + File.separator;
```

Comment:
Move 'String dataUploadDir = appService.getFileUploadDir();' above 'this.testCompanyField(company)' as we don' t need to check anything else if uploadDir is null
And check with append File.separator in same line of code, no need of:
'dataUploadDir = dataUploadDir + File.separator;'